Makefile

```diff
@@ -123,7 +123,7 @@ vanity-import-fix: $(PORTO)
 # Generate go.work file for local development.
 .PHONY: go-work
 go-work: $(CROSSLINK)
-	$(CROSSLINK) work --root=$(shell pwd)
+	$(CROSSLINK) work --root=$(shell pwd) --go=1.22.7
```

Comment:
Should we document to change this when 1.22 is EOL (which shouldn't be too far away, since 1.24 is expected this month)?

Reply:
Any idea where to document it?

Reply:
Add a note to releasing.md?

Reply:
Maybe expand https://github.com/open-telemetry/opentelemetry-go/issues/5870

Reply:
> Maybe expand https://github.com/open-telemetry/opentelemetry-go/issues/5870

Done